index.js

```diff
@@ -1,12 +1,47 @@
-const mongoose = require('mongoose');
-const Recipe = require('./models/Recipe'); // Import of the model Recipe from './models/Recipe'
-const data = require('./data.js');  // Import of the data from './data.js'
+const mongoose = require("mongoose");
+const Recipe = require("./models/Recipe"); // Import of the model Recipe from './models/Recipe'
+const data = require("./data.js"); // Import of the data from './data.js'
 
 // Connection to the database "recipeApp"
-mongoose.connect('mongodb://localhost/recipeApp', { useNewUrlParser: true })
+mongoose
+  .connect("mongodb://localhost/recipeApp", {
+    useNewUrlParser: true
+  })
   .then(() => {
-    console.log('Connected to Mongo!');
-  }).catch(err => {
-    console.error('Error connecting to mongo', err);
-  });
-
+    return Recipe.create({
+      title: "Chocolat",
+      level: "Easy Peasy",
+      ingredient: ["eggs", "cheese"],
+      cuisine: "French",
+      dishType: "Breakfast",
+      duration: 25,
+      creator: "Sarah Le Du"
+    });
+  })
+  .then(cookie => {
+    console.log("recipe added");
+    console.log(cookie);
+    return Recipe.insertMany(data);
+  })
+  .then(list => {
+    list.forEach(l => console.log(l.title));
+    return Recipe.updateMany(
```

Comment:
when you need to only update one and if you have the id, you can use model.findByIdAndUpdate(id, {duration:100})